larray/tests/test_array.py

```diff
@@ -3534,6 +3534,13 @@ def test_0darray_convert(self):
         expected_np12 = "only integer scalar arrays can be converted to a scalar index"
         assert msg in {expected_np11, expected_np12}
 
+    def test_deprecated_methods(self):
+        with pytest.warns(FutureWarning):
```

Comment:
should check the actual message and that the warning "points" to the correct file with something like:
```python
        with pytest.warns(RuntimeWarning) as caught_warnings:
            ...
        assert len(caught_warnings) == 1
        warn_msg = "..."
        assert caught_warnings[0].message.args[0] == warn_msg
        assert caught_warnings[0].filename == __file__
```